docs/en/setup/plugins/client_grpc-client.md

```diff
@@ -30,6 +45,7 @@ check_period: 5
 ## Configuration
 |Name|Type|Description|
 |----|----|-----------|
+| finder_type | string | The gRPC server address finder type |
```

Comment:
Please describe how to activate different finders.

Reply:
been added to the example file.